### DISCUSSION TITLE
Is Calendar pure JS ?  is it using Jquery?

### DISCUSSION THREAD

**Discussion by 99bits** [nhn/tui.calendar#1121] [General]
Hi,

Love this calendar, is this pure JS, reason is that I saw reference to JQuery.

**Comment by adhrinae** [comment]
Yes, It's written in the Vanilla JS.

https://github.com/nhn/tui.calendar/blob/6d2f0d47fb012484b84ba3ecf4db1631bf70cfe1/package.json#L105-L110

The reason for mentioning jQuery is that It's a pretty old library and it was common sense to support the jQuery plugin for most of the users using jQuery.

https://github.com/nhn/tui.calendar/blob/6d2f0d47fb012484b84ba3ecf4db1631bf70cfe1/src/index.js#L13-L38
